stix_shifter/requirements.txt

```diff
@@ -12,3 +12,5 @@ xmltodict==0.13.0
 jsonmerge==1.8.0
 colorlog==6.6.0
 stix2-matcher==3.0.0
+ipaddress==1.0.23
```

Comment:
ipaddress is part of standard python https://docs.python.org/3.9/library/ipaddress.html

Reply:
Removed